packages/terra-form-input/src/terra-dev-site/doc/common/BlankExample.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Input from '../../../Input';
+import Input from 'terra-form-input'
```

Comment:
@mjhenkes Does this lose hot reloading by defining the module?

Reply:
I believe so.

Reply:
Talked with @mjhenkes and the current format is used to enable hot reloading. Changing the format is something we're not interested in updating at this time.

Reply:
Thanks @bjankord. It appears most of your other examples define the module instead of using a relative import. Could you explain the discrepancy? Does `terra-form-input` have a special need for hot reloading that other examples don't?

Reply:
The other examples likely haven't been updated to a format that supports hot-reloading with the webpack config used for this repo.